renovate.json

```diff
@@ -57,6 +57,10 @@
     {
       "packagePatterns": ["wdio", "webdriver"],
       "groupName": "webdriverio packages"
+    },
+    {
+      "packagePatterns": ["nightwatch"],
```

Comment:
Should `chromedriver` be included ?

Reply:
yes, it needs at least one driver to run tests, `chromedriver` is the most common one.

Reply:
@clarkdo sorry I replied too fast here! I've just added chromedriver in renovate.json, I forgot to add it before!